bindings/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,790 @@
+  * [Adding a new function](#adding-a-new-function)
+  * [Adding a new module](#adding-a-new-module)
+  * [Adding a new class](#adding-a-new-class)
+  * [Adding or modifying the function's signature](#adding-or-modifying-the-functions-signature)
+* [Advanced modifications of functions](#advanced-modifications-of-functions)
+  * [Declaring function with clashing name](#declaring-function-with-clashing-name)
+  * [Declaring function returning complex type](#declaring-function-returning-complex-type)
+* [Advanced modifications of classes](#advanced-modifications-of-classes)
+  * [Declaring class methods](#declaring-class-methods)
+  * [Declaring class attributes, getters and  setters](#declaring-class-attributes-getters-and--setters)
+  * [Declaring class with clashing name](#declaring-class-with-clashing-name)
+  * [Declaring class usable in other modules](#declaring-class-usable-in-other-modules)
+  * [Declaring functions for string representation of objects](#declaring-functions-for-string-representation-of-objects)
+  * [Declaring functions for comparison of objects](#declaring-functions-for-comparison-of-objects)
+* [Other advanced features](#other-advanced-features)
+  * [Dereferencing and retyping the object](#dereferencing-and-retyping-the-object)
+    * [Dereferencing shared pointer](#dereferencing-shared-pointer)
+    * [Dereferencing iterator](#dereferencing-iterator)
+  * [Using null pointer](#using-null-pointer)
+  * [Defining C variables in code](#defining-c-variables-in-code)
+  * [Working with standard streams](#working-with-standard-streams)
+  * [Working with iterators](#working-with-iterators)
+  * [Defining type definition](#defining-type-definition)
+  * [Calling functions of shared pointer object](#calling-functions-of-shared-pointer-object)
+  * [Supporting Python for loops](#supporting-python-for-loops)
+  * [Working with strings](#working-with-strings)
+  * [Helping Cython to infer types](#helping-cython-to-infer-types)
+  * [Using pre- and post-increments (`var++` and `++var`)](#using-pre--and-post-increments-var-and-var)
+  * [Working with exceptions](#working-with-exceptions)
+* [Importing between modules](#importing-between-modules)
+  * [Importing C/C++ defined functions and symbols](#importing-cc-defined-functions-and-symbols)
+  * [Importing Python defined functions and symbols](#importing-python-defined-functions-and-symbols)
+  * [Importing standard containers and types](#importing-standard-containers-and-types)
+* [Troubleshooting](#troubleshooting)
+  * [An error when importing Python object from other module](#an-error-when-importing-python-object-from-other-module)
+  * [A weird error when Cython cannot retype C/C++ object to Python object](#a-weird-error-when-cython-cannot-retype-cc-object-to-python-object)
+* [Quick glossary](#quick-glossary)
+<!-- TOC -->
+
+## File/Directory Structure
+
+There are two types of files:
+
+- `*.pxd`: Cython files containing declarations/definitions of the C/C++ API.
```

Comment:
Append: "specifying what you want to import from the C/C++ API"

Reply:
PS: More comments to come. Experimenting with external PR review workflow.

Reply:
Sure. Btw. for short rewording or grammar fixes, please, prefer suggestions, so I can directly commit the changes whenever possible.